### PR TITLE
Qu auto/sorqa 125

### PR DESCRIPTION
**Pull request by pk-sgent**

### CODE REVIEW

**Comment by pk-sgent** [comment]
image

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java [hunk 1]
```diff
@@ -148,8 +198,48 @@ public CustomCaseExportCSV parseCustomCaseExport(String fileName) {
       builder =
           CustomCaseExportCSV.builder()
               .disease(values[0])
-              .firstName(values[1])
-              .lastName(values[2])
+              .firstName(String.format(values[1], Locale.GERMAN))
+              .lastName(String.format(values[2], Locale.GERMAN))
+              .build();
+    } catch (NullPointerException e) {
+      log.error("Null pointer exception parseCustomCaseExport: ", e);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java:205]: you need to add the {} so the string will be formatted and print e value ( also use e.getMessage() )

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java [hunk 2]
```diff
@@ -148,8 +198,48 @@ public CustomCaseExportCSV parseCustomCaseExport(String fileName) {
...
+    Case builder = null;
+    CSVParser csvParser = new CSVParserBuilder().withSeparator(';').build();
+    try (CSVReader reader =
+        new CSVReaderBuilder(new FileReader(fileName))
+            .withCSVParser(csvParser)
+            .withSkipLines(2) // parse only data
+            .build()) {
+      r = reader.readAll();
+    } catch (IOException e) {
+      log.error("IOException parseCustomCaseExport: ", e);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java:224]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java [hunk 3]
```diff
@@ -148,8 +198,48 @@ public CustomCaseExportCSV parseCustomCaseExport(String fileName) {
...
+    } catch (CsvException e) {
+      log.error("CsvException parseCustomCaseExport: ", e);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java:226]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java [hunk 4]
```diff
@@ -102,20 +116,56 @@ public CaseImportExportSteps(
...
+              String.format(apiState.getLastCreatedPerson().getLastName(), Locale.GERMAN),
+              "Last names are not equal");
+          softly.assertAll();
+        });
+
+    When(
+        "I check if downloaded data generated by basic case option is correct",
+        () -> {
+          String file =
+              "./downloads/sormas_f\u00E4lle_" + LocalDate.now().format(formatter) + "_.csv";
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java:143]: are we sure that the time difference is ok? after file is downloaded and we reach this line, at least 1ms will pass over for sure

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java (hunk 4 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java:143]: Yes, there is not time but date :)

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**